### ISSUE TITLE
Support for capturing template result

### ISSUE THREAD

**Issue by Tieno** [slimphp/Slim#233]
I'd be nice if the Slim app instance could provide direct support for capturing the render result too, instead of directly outputting it. (I use some twig templates for sending emails).

So far I've done this myself by adding a method to the TwigView extension class, which is accessed like so $app->view()->getRender('template.twig', $data);

**Comment by pendexgabo** [comment]
This feature already exists. Check the method on https://github.com/codeguy/Slim/blob/master/Slim/View.php#L153 you just need to call (AFAIK)

``` php
$app->view()->setData('color', 'red');
$result = $app->view()->render('template.twig');
```

**Comment by codeguy** [comment]
Closing. This was solved in a previous pull request. Will be available on the View object in 1.6.3.
